### ISSUE TITLE
NTR: Aspartylglycosaminidase low in fibroblasts or leukocytes (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8161]
New term request Aspartylglycosaminidase low in fibroblasts or leukocytes (f421): N-aspartyl-beta-glucosaminidase

**Comment by pnrobinson** [comment]
@MickeySegal

There may be a typo here, do we mean Aspartylglucosaminidase (AGA; [EC 3.5.1.26](https://enzyme.expasy.org/EC/3.5.1.26))

"u" and not "y"

https://omim.org/entry/613228?search=aspartylglucosaminidase&highlight=aspartylglucosaminidase

**Comment by MickeySegal** [comment]
I see in https://omim.org/entry/208400 that both gly and glu terms have been used, but the glu form seems to be the preferred form now.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000573
- label: Reduced tissue aspartylglucosaminidase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:1904874
- def: Activity of aspartylglucosaminidase (AGA; EC 3.5.1.26) in the tissues below the lower limit of normal. AGA activity can be measured in multiple tissues including leukocytes and cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8161
- synonyms:
